module/common/SystemInfo.h

```diff
@@ -1,13 +1,14 @@
 /**
  * @file SystemInfo.h
  * @brief 走行システムで統一する情報をまとめたファイル
- * @author takahashitom
+ * @author takahashitom keiya121
  */
 
 #ifndef SYSTEM_INFO_H
 #define SYSTEM_INFO_H
 
 static constexpr double RADIUS = 50.0;  // 車輪の半径[mm]
 static constexpr double TREAD = 125.0;  // 走行体のトレッド幅（両輪の間の距離）[mm]
+static bool shouldGetRunLog = false;
```

Comment:
特に明確な区別がなければ、LogかLogsに統一した方がいいかな。

```suggestion
static bool shouldGetRunLogs = false;
```

Reply:
RunLogsに統一しました。ありがとうございます。